src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -49,8 +51,17 @@ export const DatasetSearchTable = (props) => {
   const [selectedTable, setSelectedTable] = useState(datasetSearchTableTabs.study);
   const [searchTerm, setSearchTerm] = useState('');
 
-  const isFiltered = (filter, category) => (filters[category]).indexOf(filter) > -1;
-  const numSelectedFilters = (filters) => Object.values(filters).reduce((sum, array) => sum + array.length, 0);
+  const isFilteredArray = (filter, category) => (filters[category]).indexOf(filter) > -1;
+
+  const anyFiltersSelected = (filters) => {
+    return Object.values(filters).some((filter) => {
```

Comment:
For a single line lambda, you can omit the `{ return }`
```suggestion
  const anyFiltersSelected = (filters) =>
    Object.values(filters).some((filter) => {
```

I know in terra-ui this was the standard, not sure if duos-ui has an opinion about this.